docs/sources/api/_index.md

```diff
@@ -209,37 +217,37 @@ $ curl -G -s  "http://localhost:3100/loki/api/v1/query" --data-urlencode 'query=
   }
 }
 ```
+Set the `x-scope-orgid` header,
```

Comment:
```suggestion
Set the `X-Scope-OrgID` header,
```
Header field names should be case-insensitive but this is how we use it everywhere.